src/github/issue.ts

```diff
@@ -28,7 +28,9 @@ export const issueWebhookHandler = async (context: WebhookContext<WebhookPayload
 	// TODO: need to create reusable function for unfurling
 	let linkifiedBody;
 	try {
-		linkifiedBody = await util.unfurl(issue.body);
+		if (issue.body !== null) {
+			linkifiedBody = await util.unfurl(issue.body);
+		}
```

Comment:
This is the only non-typescript change b/c the latest TS types allow `issue.body` to be null